ui/app/components/clients/page/counts.ts

```diff
@@ -122,30 +101,16 @@ export default class ClientsCountsPageComponent extends Component<Args> {
     return namespace ? activity.byNamespace.find((ns) => ns.label === namespace) : null;
   }
 
-  get filteredActivity() {
-    // return activity counts based on selected namespace and auth mount values
-    const { namespace, authMount, activity } = this.args;
-    if (namespace) {
-      return authMount
-        ? this.activityForNamespace?.mounts.find((mount) => mount.label === authMount)
-        : this.activityForNamespace;
-    }
-    return activity.total;
-  }
-
   @action
   onDateChange(dateObject: { dateType: string; monthIdx: string; year: string }) {
     const { dateType, monthIdx, year } = dateObject;
-    const {
-      currentTimestamp,
-      config: { billingStartTimestamp },
-    } = this.args;
+    const { currentTimestamp, config } = this.args;
     // converts the selectedDate to unix timestamp for activity query
     const selectedDate = formatDateObject({ monthIdx, year }, dateType === 'endDate');
 
     if (dateType !== 'cancel') {
       const start_time = {
-        reset: getUnixTime(billingStartTimestamp), // clicked 'Current billing period' in calendar widget -> resets to billing start date
+        reset: getUnixTime(config?.billingStartTimestamp) || null, // clicked 'Current billing period' in calendar widget -> resets to billing start date
```

Comment:
null makes the QP disappear from the URL if a user doesn't have the billing start date info. (undefined caused an error)